Detections/ASimDNS/imDns_DomainEntity_DnsEvents.yaml

```diff
@@ -0,0 +1,69 @@
+id: 999e9f5d-db4a-4b07-a206-29c4e667b7e8
+name: (Preview) TI map Domain entity to Dns Event (Normalized DNS)
+description: |
+  'Identifies a match in DNS events from any Domain IOC from TI
```

Comment:
Do we need the single quotes? does it work in multi-line? Also, need to end sentences with a full stop (".")